colink/instant_server.py

```diff
@@ -29,6 +29,10 @@ def new():
                     "-c",
                     'bash -c "$(curl -fsSL https://raw.githubusercontent.com/CoLearn-Dev/colinkctl/main/install_colink.sh)"',
                 ],
+                env={
```

Comment:
apply formatter?

Reply:
Here I already applied the formatter and it is fine.